apps/vs-code-designer/src/app/commands/createCodeful/CodefulWorkflowCreateStep.ts

```diff
@@ -0,0 +1,139 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import type { IActionContext } from '@microsoft/vscode-azext-utils';
+import { nonNullProp } from '@microsoft/vscode-azext-utils';
+import { WorkflowProjectType, MismatchBehavior, WorkerRuntime } from '@microsoft/vscode-extension-logic-apps';
+import type { IFunctionWizardContext, IHostJsonV2 } from '@microsoft/vscode-extension-logic-apps';
+import { writeFileSync } from 'fs';
+import * as fse from 'fs-extra';
+import * as path from 'path';
+import { WorkflowCreateStepBase } from '../createCodeless/createCodelessSteps/WorkflowCreateStepBase';
+import { getCodefulWorkflowTemplate } from '../../utils/codeless/templates';
+import { writeFormattedJson } from '../../utils/fs';
+import {
+  hostFileName,
+  extensionBundleId,
+  defaultVersionRange,
+  azureWebJobsStorageKey,
+  localEmulatorConnectionString,
+  workerRuntimeKey,
+  functionsInprocNet8Enabled,
+  codefulWorkflowFileName,
+  appKindSetting,
+  logicAppKind,
+  functionsInprocNet8EnabledTrue,
+} from '../../../constants';
+import { setLocalAppSetting } from '../../utils/appSettings/localSettings';
+import { validateDotnetInstalled } from '../../utils/dotnet/executeDotnetTemplateCommand';
+import { switchToDotnetProject } from '../workflows/switchToDotnetProject';
+import * as vscode from 'vscode';
+import { createConnectionsJson } from '../../utils/codeless/connection';
+import { createEmptyParametersJson } from '../../utils/codeless/parameter';
+
+export class CodefulWorkflowCreateStep extends WorkflowCreateStepBase<IFunctionWizardContext> {
```

Comment:
In future PRs I want to investigate how to simplify this between our three kinds of "CreateStep"s as they all have pieces in common